### PR TITLE
Feat: Added Mobile No. field to Client Details Screen

### PR DESCRIPTION
**Pull request by moksh-mahajan**
photo6318831305957157482

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by moksh-mahajan** [comment]
@tarun0 Please review this PR

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java [hunk 1]
```diff
@@ -117,6 +117,9 @@
     @BindView(R.id.tv_office)
     TextView tv_office;
 
+    @BindView(R.id.tv_mobile_no)
+    TextView tv_mobile_no;
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: Correct naming convention

File: mifosng-android/src/main/java/com/mifos/objects/client/Client.java [hunk 2]
```diff
@@ -110,6 +112,14 @@ public void setGroupId(int groupId) {
         this.groupId = groupId;
     }
 
+    public String getMobileNo() {
+        return mobileNo;
+    }
+
+    public void setMobileNo(String mobileNo) {
+        this.mobileNo = mobileNo;
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/objects/client/Client.java]: If mobile number is not present, provide some default String like 'Not Available'.

Comment (moksh-mahajan) [reply mifosng-android/src/main/java/com/mifos/objects/client/Client.java]: If the mobile number is not present then mobile no. TableRow will be hidden. ```(TextUtils.isEmpty(client.getMobileNo())) rowMobileNo.setVisibility(GONE);```

**Comment by moksh-mahajan** [comment]
@tarun0 I have made the requested changes. Please review.

**Pull request merged**